airbyte-integrations/connectors/source-square/integration_tests/configured_catalog.json

```diff
@@ -143,18 +143,6 @@
       "sync_mode": "full_refresh",
       "cursor_field": ["id"],
       "destination_sync_mode": "overwrite"
-    },
-    {
```

Comment:
orders stream is temporary removed from tests because it is not working due to bug in CDK